### ISSUE TITLE
[Feature Request]: Add keybinding for the "Switch workspace" action

### ISSUE THREAD

**Issue by Reyhn3** [ForkIssues/TrackerWin#2186]
I'd like to have a keybinding for the "Switch workspace" action.

The keybinding should show the same dialog that is displayed **after** I show the quick launch window `ctrl + p`, type `switch` and press `enter`.

Suggested keybinding for the switch-dialog:
`ctrl + s` 

Additional keybindings:
`alt + 1` to immediately switch to the first workspace, `alt + 2` to switch to the second etc.

**Comment by DanPristupov** [comment]
> `ctrl + s`

It's used by stage

> `alt + 1`

`alt` keybindings are reserved by Windows.

**Comment by Reyhn3** [comment]
> > `ctrl + s`
> 
> It's used by stage

Really? `ctrl+s` is already in use? I couldn't find it under `Help -> Keyboard Shortcuts` (https://github.com/fork-dev/TrackerWin/issues/333). Nor in any tooltip.
Then just change it to something that is available, like `F4` or anything. The point is to have a keybinding for it.



> > `alt + 1`
> 
> `alt` keybindings are reserved by Windows.

What do you mean?
`alt + 1` is not reserved by Windows. Maybe you're thinking of `win + 1`? I use the alt + number in other applications, like Rider, Excel, komorebi etc. Anyways, each workspace could have it's own keyboard shortcut, like `ctrl + shift + 1`, `alt + shift + 1` etc. Let me know if you'd like me to report this keybinding in a separate issue.

**Comment by DanPristupov** [comment]
> Really? `ctrl+s` is already in use?

Yes. The context menu shows only 'Enter', because it's not possible to show 2 shortcuts.

> > alt keybindings are reserved by Windows.
> 
> What do you mean?

Guidelines for Keyboard User Interface Design: https://learn.microsoft.com/en-gb/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design?redirectedfrom=MSDN#assigning-shortcut-keys

> Avoid ALT+letter combinations because they may conflict with access keys.

In addition to this, Workspaces dropdown is not a flat list, it supports folders. Assigning Key + number combination is not right.

If I remember correctly, you can use Ctrl+Click to switch workspace to the next one.

**Comment by Reyhn3** [comment]
Ah, I see. You're referring to the access keys. Well, that's no impediment either, because you as a developer control whether or not you want to use access keys. Access keys are not created automatically - they are intentionally added by a developer. It is only a guideline either way.

Yes, `ctrl+click` on the workspace button cycles between workspaces.
This, and the stage-shortcut, should be documented at the top of #333.

But all of this is beside the point, which is that I'd like to have a keybinding for getting the "switch workspace" dialog up directly (without having to go via the command dialog and search for the action).

**Comment by havrlisan** [comment]
Writing here as it's suitable instead of writing a new issue: it would be nice also to have <kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>Click</kbd> for cycling the workspaces in reverse.

**Comment by Amparose** [comment]
As I'm sure this will impact MacOS users, instead of all this nonsense on hard-coding a shortcut, can you just add it to the main menu and then everyone can not only set their own shortcut to suit (not sure if ppl suffering windows can do that) but also use it in automation/Keyboard Maestro.
